spec/status.xml

```diff
@@ -5,62 +5,47 @@
+href="https://www.w3.org/TR/">W3C technical reports index</loc> at
+https://www.w3.org/TR/.</emph></p>
 
-<p>This document has been reviewed by W3C Members, by software developers,
-and by other W3C groups and interested parties, and is endorsed by the Director
-as a W3C Recommendation. It is a stable document and may be used as reference
-material or cited from another document. W3C's role in making the Recommendation
-is to draw attention to the specification and to promote its widespread
-deployment. This enhances the functionality and interoperability of the Web.</p>
+<p>This document was published by the <loc href="http://www.w3.org/AudioVideo/TT/">Timed Text Working Group</loc> as a Candidate Recommendation. 
+This document is intended to become a W3C Recommendation. Comments regarding this document are welcome. 
+Please send them to <loc href="mailto:public-tt@w3.org?subject=%5Bttml1%5D">public-tt@w3.org</loc> 
+(<loc href="mailto:public-tt-request@w3.org?subject=subscribe">subscribe</loc>,
+<loc href="https://lists.w3.org/Archives/Public/public-tt/">archives</loc>) with <code>[ttml1]</code> at the start of your email's subject.
+W3C publishes a Candidate Recommendation to indicate that the document is believed to be stable 
+and to encourage implementation by the developer community. This Candidate Recommendation is expected to 
+advance to Proposed Recommendation no earlier than &crEnd;.</p>
 
-<p>This version reflects changes made as a result of applying errata and addressing
-comments received since the Recommendation was first published. With one minor
-exception noted below, these changes are editorial.</p>
+<p>This new edition of the Timed Text Markup Language (TTML) 1.0
+<bibref ref="ttml10"/> clarifies ambiguities and corrects errors identified in the 
+<loc href="&prevrec;">previous
+edition</loc>, which this new edition supersedes. No features were added.</p>
 
-<p>This document is a revision of Timed Text Markup Language (TTML) 1.0
-<bibref ref="ttml10"/>, superseding that <loc href="http://www.w3.org/TR/2010/REC-ttaf1-dfxp-20101118/">previous
-Recommendation</loc>. The only identified substantive difference between this version and
-its predecessor is a change of the default treatment of the <att>ttp:markerMode</att> parameter attribute
-from <code>continuous</code> to <code>discontinuous</code>. A cumulative summary of all changes applied
-to this version since the Recommendation was first published is available at
-<loc  href="ttml1-changes.html#change-history-rec-to-2e-per">Changes
-from Recommendation to Proposed Edited Recommendation (2nd Ed.)</loc> and
-<loc  href="ttml1-changes.html#change-history-2e-per-to-rec-2e">Changes
-from Proposed Edited Recommendation (2nd Ed.) to Recommendation (2nd Ed.)</loc>.</p>
+<p>Substantive changes applied since the <loc href="&prevloc;">previous version of this specification</loc> are listed at 
+<loc href="substantive-changes-summary.txt">substantive-changes-summary.txt</loc>.</p>
```

Comment:
This is a 404 at the moment. I suspect the `spec/substantive-changes-summary.txt` file is not copied to the built output folder by the build script.

Reply:
Fixed the build script